### ISSUE TITLE
Inner shelf

### ISSUE THREAD

**Issue by tuvabjornberg** [tuvabjornberg/bookshelf#3]
Inner (middle) shelves of the bookshelf should be extended to the edge. There is now an annoying gap to the edge.

**Comment by tuvabjornberg** [comment]
Mostly fixed in [3aba463](https://github.com/tuvabjornberg/bookshelf/commit/3aba463a596d67c9478b8007faa16e88d4fdd05c).

**Comment by tuvabjornberg** [comment]
Probably fixed itself when ListView got fixed for the book generation on shelf.
```
Container(
    color: lightBrownEdgeShelf,
    width: double.maxFinite,
    padding: const EdgeInsets.symmetric(
        horizontal: 0,
        vertical: 8,
    ),
 )
```
